crates/llm-base/src/loader.rs

```diff
@@ -57,6 +62,7 @@ impl TryFrom<i32> for FileType {
             7 => FileTypeFormat::MostlyQ8_0,
             8 => FileTypeFormat::MostlyQ5_0,
             9 => FileTypeFormat::MostlyQ5_1,
+            10 => FileTypeFormat::MostlyQ2_K,
```

Comment:
Aren't the other k_quants missing here?

Reply:
There is a `Q8_K`, but it is defined [in some places](https://github.com/ggerganov/llama.cpp/blob/master/ggml.h#L249), but [not in others](https://github.com/ggerganov/llama.cpp/blob/master/ggml.h#L278).

Reply:
Aren't `Q3_K` to `Q6_K` missing here? you implemented them in `From<FileType>` function but not in the `TryFrom<i32>` function.

Reply:
Right 👍
Fixed!